opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -489,6 +491,16 @@ class BrineCo2Pvt
         return convertXoGToRs(convertxoGToXoG(xlCO2, salinity), regionIdx);
     }
 
+    /*!
+     * \brief Get the CO2 parameters.
+     *
+     * \return A const reference to the CO2 parameters.
+     */
+    const Params& getCo2Tables() const
```

Comment:
Is this just for debugging? Can it be removed? If not, perhaps also add it to the Co2GasPvt class.